packages/app/src/components/Blank.jsx

```diff
@@ -79,142 +79,84 @@ const defaultStyles = `
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    text-align: center;
+    color: #747994;
+    margin: 0 32px;
+  }
+`
+
+const blankContentsHtml = (text = '', subtext = '') => {
   return `
     <style>
-      ${defaultStyles}
-
-      ${listStyles}
+      ${blankStyles}
     </style>
 
-    <div class='container'>
-      ${svgCy}
-      <p>This is the default blank page.</p>
-      <p>To test your web application:</p>
-      <ul>
-        <li>Mount your component with
-          <kbd>
-            <a href='https://on.cypress.io/mount' target='_blank'>mount()</a>
-          </kbd>
-        </li>
-        <li>Begin writing tests</li>
-      </ul>
+    <div class='outer-container'>
+      <div class='inner-container'>
+        ${svgCy}
+        <p class="text">${text}</p>
+        <p class="subtext">${subtext}</p>
+      </div>
     </div>
   `
 }
 
-export const sessionLifecycle = () => {
-  return `
-    <style>
-
-      ${defaultStyles}
-
-      ${listStyles}
-
-      .warn {
-        color: #bd5800;
-        background-color: #f5f4d7;
-        font-size: 15px;
-        font-weight: normal;
-        padding: 16px;
-        max-width: 600px;
-        margin: 0 16px 8px;
-        display: inline-block;
-        text-align: left;
-        width: 90%;
-      }
-
-      .em {
-        font-style: italic;
-      }
-    </style>
+export const initial = () => {
+  return blankContentsHtml()
+}
 
-    <div class='container'>
-      ${svgCy}
-      <br/>
-      <p class="warn">Because <code><b>experimentalSessionAndOrigin</b></code> is enabled, Cypress navigates to the default blank page <span class="em">before each test</span> to ensure test reliability.</p>
-      <p>This is the default blank page.</p>
-      <p>To test your web application:</p>
-      <ul>
-        <li>Start your app's server</li>
-        <li>
-          <kbd>
-            <a href='https://on.cypress.io/visit' target='_blank'>cy.visit()</a>
-          </kbd>
-          your app
-        </li>
-        <li>Begin writing tests</li>
-      </ul>
-    </div>
-  `
+export const sessionLifecycle = () => {
+  return blankContentsHtml('Default blank page',
+    'This page was cleared by navigating to about:blank.<br>All active session data (cookies, localStorage and sessionStorage) across all domains are cleared.')
 }
 
 export const session = () => {
-  return `
-    <style>
-      ${defaultStyles}
-
-      ${listStyles}
-
-    </style>
-
-    <div class='container'>
-    ${svgCy}
-      <p>This is a blank page.</p>
-      <p>We always navigate you here after <kbd>cy.session()</kbd></p>
-      <p>To continue your test, follow up the command with
-      <kbd><a href='https://on.cypress.io/visit' target='_blank'>cy.visit()</a></kbd>
-      </p>
-<pre><code>
-1 |
-2 | cy.session(...)
-3 | <b>cy.visit(...)</b>
-4 |
-</code></pre>
-
-    </div>
-  `
+  return blankContentsHtml('Default blank page', 'This page was cleared by navigating to about:blank.')
```

Comment:
does it make sense to put `Default blank page` and `This page was cleared by navigating to about:blank` in  constant variables above to inject via reference so we don't need to change the text in two places in the future if needed?

Reply:
added variables: [c27e256](https://github.com/cypress-io/cypress/pull/23095/commits/c27e256dce8e30805999de4ae2c61182a13b59d4)